### ISSUE TITLE
[meta] API should be there!

### ISSUE THREAD

**Issue by KengoTODA** [partakein/partake#6]
API to manage events is required.<p>Migrated from http://code.google.com/p/partakein/issues/detail?id=6</p><hr/><h2>earlier comments</h2>

<p><strong>kawanaka.shinya said, at 2010-12-24T06:20:32.000Z:</strong></p>

<p>Issue 74 has been merged into this issue.</p><p><strong>kawanaka.shinya said, at 2011-04-28T08:11:02.000Z:</strong></p>

<p>わりとテスト環境が整ってきたので、API 系はテストがないものは認めないぐらいの気分でテストを書いていきたいと思います。
</p><p><strong>mayah@mayah.jp said, at 2012-01-12T14:14:37.000Z:</strong></p>

<p>bulk edit</p><p><strong>mayah@mayah.jp said, at 2012-03-04T03:54:52.000Z:</strong></p>

<p>Partake beta2 has a lot of APIs, including:
  Creating/Modifying/Deleting events,
  Enrolling/Reserving/Canceling events,
  Geting enrolled events, and so on.

Basically all things you can do in PARTAKE web page have API, because PARTAKE webpage also basically uses that API.</p>

**Comment by mayah** [comment]
We have a lot of APIs now. CLOSED.
